Code/Framework/AzFramework/AzFramework/Spawnable/InMemorySpawnableAssetContainer.h

```diff
@@ -56,13 +44,23 @@ namespace AzToolsFramework::Prefab::PrefabConversionUtils
         SpawnableAssets&& MoveAllInMemorySpawnableAssets();
         const SpawnableAssets& GetAllInMemorySpawnableAssets() const;
 
+        //! Creates an in-memory spawnable asset given a list of product asset data
+        CreateSpawnableResult CreateInMemorySpawnableAsset(
+            AssetDataInfoContainer& assetDataInfoContainer,
+            bool loadReferencedAssets,
+            const AZStd::string& rootSpawnableName);
+
+        //! Creates an in-memory spawnable asset given a single spawnable.
+        //! Ex. usage is when a network server receives in-memory spawnable data from the client
+        CreateSpawnableResult CreateInMemorySpawnableAsset(
```

Comment:
repeated: For any newly added functions in this PR, I recommend using @param and @return tags to explain the parameters and return values as well